taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -314,12 +356,20 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
                     )
                 } )
             }
-            {/* Dialog box for adding translations */}
+            {/* Button to add language to the list of shown languages or show the language if it already exists */}
+            <Button sx={{mt: 2}} onClick={handleOpen}>
+                Add/Show More Languages 
+                ({
+                    addedLanguageCodes.length - 
+                    addedLanguageCodes.filter(languageCodeItem => shownTranslationLanguages.includes(languageCodeItem)).length
+                } Hidden)
+            </Button>
+            {/* Dialog box for adding languages to the list of shown languages */}
             <Dialog open={openDialog} onClose={handleClose}>
-                <DialogTitle>Add a language</DialogTitle>
+                <DialogTitle>Add or Show another language</DialogTitle>
                 <DialogContent>
                 <DialogContentText>
-                    Enter the two letter language code for the language to be added.
+                    Enter the two letter language code for the language to be shown or added.
```

Comment:
Is this a valid message? It seems to me that we can just remove it

Reply:
Yup, seems redundant.